admin/class-yoast-plugin-conflict.php

```diff
@@ -306,6 +306,23 @@ protected function find_plugin_category( $plugin ) {
 		}
 	}
 
+	/**
+	 * Get plugin name from file.
+	 *
+	 * @param string $plugin Plugin path relative to plugins directory.
+	 *
+	 * @return string|bool
+	 */
+	protected function get_plugin_name( $plugin ) {
+		$plugin_details = get_plugin_data( WP_PLUGIN_DIR . '/' . $plugin );
+
+		if ( $plugin_details['Name'] !== '' ) {
+			return $plugin_details['Name'];
+		}
+
+		return false;
```

Comment:
Is there a specific reason that this method also returns a boolean and not an empty string?

Reply:
Just moved it and did not change because I don't want to break the code 😬